Simplified/Reader2/NYPLReaderBookmarksBusinessLogic.swift

```diff
@@ -20,6 +20,9 @@ class NYPLReaderBookmarksBusinessLogic: NSObject, NYPLReadiumViewSyncManagerDele
   private let drmDeviceID: String?
   private let bookRegistryProvider: NYPLBookRegistryProvider
   private let currentLibraryAccountProvider: NYPLCurrentLibraryAccountProvider
+  private var lastReadPositionUploadDate: Date
+  private var queuedReadPosition: String = ""
+  private let serialQueue = DispatchQueue(label: "\(Bundle.main.bundleIdentifier!).bookmarkBusinessLogic")
```

Comment:
a performance improvement here would be to specify the `target` param to be one of the global queues. This optimizes thread handling on the OS, i.e. preventing iOS to needlessly generate new threads.